frontend/lib/laletterbuilder/letter-builder/habitability/routes.tsx

```diff
@@ -52,6 +54,7 @@ export const getHabitabilityProgressRoutesProps = (): ProgressRoutesProps => {
   const routes = LaLetterBuilderRouteInfo.locale.habitability;
 
   return {
+    label: li18n._(t`Build your Letter`),
```

Comment:
@samaratrilling Here is the text that goes on top of the progress bar for the whole habitability flow! It'll look like: `Build your Letter: Step X of 10`